sdk/src/impl/position-impl.ts

```diff
@@ -40,87 +45,118 @@ export class PositionImpl implements Position {
   async increaseLiquidity(
     liquidityInput: IncreaseLiquidityInput,
     sourceWallet?: Address,
-    positionWallet?: Address
+    positionWallet?: Address,
+    payer?: PublicKey
```

Comment:
nit: I think this might be a little ambiguous here, since it's only used for paying for the ATAs and not for the any other parts of the transaction?

Reply:
ah. good catch. increase_liquidity has no use for payer. I forgot to remove it when i removed resolveATA from increase-liq

Reply:
For decrease-liquidity, i've renamed it to ataPayer for better clarity